e2e/jest.test-setup.ts

```diff
@@ -64,9 +70,26 @@ beforeEach(async () => {
        : null;
   }
 
-  // Api "/workspaces" or "/cdrVersions" response can be truncated
-  const isWorkspacesApi = (request: Request): boolean => {
-    return request && (request.url().endsWith('/v1/workspaces') || request.url().endsWith('/v1/cdrVersions'));
+  // Api response won't be logged.
+  const skipApiResponseBody = (request: Request): boolean => {
```

Comment:
naming nit: shouldSkipApiResponseBody, this method doesn't do any of the skipping itself.

Reply:
👍